packages/vuetify/src/components/VBtn/VBtn.tsx

```diff
@@ -113,6 +113,14 @@ export const VBtn = defineComponent({
       return props.variant === 'elevated' && !(props.disabled || props.flat || props.border)
     })
 
+    const valueAttr = computed((): string | undefined => {
+      if (props.value === undefined) {
+        return undefined
+      }
+
+      return typeof props.value === 'string' ? props.value : JSON.stringify(props.value, null, 0)
+    })
+
```

Comment:
I can inline this computed property as

```ts
const valueAttr = computed((): string | undefined => typeof props.value === 'undefined'
      ? undefined : typeof props.value === 'string'
        ? props.value : JSON.stringify(props.value, null, 0))
```

not a fan of shortening a few chars and losing readability, but happy to change if better reflects the style

Reply:
if you want you could do the following to remove 2 loc

`if (props.value === undefined) return undefined`